### PR TITLE
Add models & columns

### PR DESCRIPTION
**Pull request by PadawanZem**
created stall & booking models & add first_name last_name & address column to user model

### CODE REVIEW

**Comment by PadawanZem** [comment]
@cameliaFauchart @Trixkiddo 
Can you take a look and confirm if you're ok with what has been done?

File: db/migrate/20211123095229_create_bookings.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class CreateBookings < ActiveRecord::Migration[6.0]
+  def change
+    create_table :bookings do |t|
+      t.string :status
+      t.datetime :start_date
+      t.datetime :end_start
```
Comment (Trixkiddo) [inline db/migrate/20211123095229_create_bookings.rb:6]: Vous me ferez 3 Ave Maria : *end_date

File: db/schema.rb [hunk 2]
```diff
@@ -10,11 +10,34 @@
...
-ActiveRecord::Schema.define(version: 2021_11_22_155953) do
+ActiveRecord::Schema.define(version: 2021_11_23_100127) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "bookings", force: :cascade do |t|
+    t.string "status"
+    t.datetime "start_date"
+    t.datetime "end_start"
```
Comment (Trixkiddo) [inline db/schema.rb:21]: Same sentence (end_date)

**Review by Trixkiddo — APPROVED** [review]
Thank you guys. Please check the 2 small errors.

**Pull request merged**